### ISSUE TITLE
2.16 bug: Reviewer cannot be cast to DeckPicker

### ISSUE THREAD

**Issue by BrayanDSO** [ankidroid/Anki-Android#13796]
```
java.lang.ClassCastException: com.ichi2.anki.Reviewer cannot be cast to com.ichi2.anki.DeckPicker
	at com.ichi2.anki.dialogs.DialogHandler.handleMessage(DialogHandler.kt:49)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7941)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:553)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
```
Blackview Android 12 2.16beta01
https://ankidroid.org/acra/app/1/bug/47657/report/dce69993-ea93-42d1-a377-efbf7c5e982f

**Comment by mikehardy** [comment]
This is likely not a regression - the intent handler and message handler both do some raw casting that blows up semi-frequently, I've seen errors like this for multiple releases over years

Should be fixed, but not going to peg this as a 2.16 blocker

**Comment by david-allison** [comment]
Closing as duplicate of https://github.com/ankidroid/Anki-Android/issues/5075
